### ISSUE TITLE
General - Request for information

### ISSUE THREAD

**Issue by MiraRatajczak** [eForms/eForms#81]
In Belgium we have created a   national Form "Request for information". This form can be used in   the stage of preliminary market consultation of the procedure.  Is it possible to introduce the possibility   in the eForms?

**Comment by rantati7** [comment]
Dear Mira,

In Finland we are using PIN for the same purpose. In our national eNotificarion system we have names this as Market Consultation but it is essentially sending PIN (without the purpose of shortening the deadline) to TED. Maybe this could be an option for you as well unless we get a specific form for this purpose.

Timo

**Comment by JachymHercher** [comment]
eForms contain two new field for informing about preliminary market consultations: BT-617 (Preliminary Consultation) and BT-117 (Preliminary Consultation Description). As @rantati7 mentions, these can be used in PINs. 

Does this meet your need? If not, could you please provide more details about what you would need?

**Comment by bouwesamplonius** [comment]
Dear Jachym,

In The Netherlands we arranged the preliminary consultation (in dutch: 'marktconsultatie') in the same way as @rantati7 describes. With the current version R2.0.9.S03 we send a F01 and put the prefix 'marktconsultatie' in the title. Adding BT-117 and BT-617 to a PIN is helping a little but does not clear a few drawbacks:

- When publishing the PIN at our national publication board or TED, the header still describes the form as a PIN. An interested party has to dive into the publication to notice BT-117 and BT-617 to conclude that this is a preliminary consultation. Technically we can fix this at a visual way so that it looks like a preliminary consultation. But at a logical datalevel it still is a PIN. This blurs reporting data at national and european level. With a separate form the data will be cleaner and much more efficient to work with. And gives less technically problems to implement.

- In some cases, the fields regarding selection criteria (BT-411, BT-72, BT-68, etcetera)) are mandatory for a PIN. But in the most cases, selection criteria are not known yet when a preliminary consultation is initiated. The same for the procedure legal basis. In The Netherlands the procedure will mostly be determined after the preliminary consultation is ended.

No surprise that we preference a separated notice type for preliminary consultation. This could be a copy of a PIN but with less fields.

**Comment by rantati7** [comment]
I do agree with you: separate PIN for market consultation would be better for the Commission statistical purposes. Also, in this stage the contract authority indeed does not know the selectition criteria - this is often in the scope of the market counsultation.

The less information one needs to fill in, the better.

**Comment by MiraRatajczak** [comment]
I agree we can use a PIN, but the reason why we have created a national form is that we have introduced a deadline to receive some answers on the consultation.

PS sorry for a late reaction but I have returned from a long holiday :-)

**Comment by JachymHercher** [comment]
Thanks a lot for the views! We have three possibilities:

1. Adding fields to existing PINs (BT-117 & BT-617).
2. Creating a new sub-type of PIN: "PIN used as a preliminary market consultation". This would be at the same level as the currently existing "PIN used as a call for competition" etc.
3. Creating a new form. 

I've spoken a bit more with colleagues dealing with innovation and their impression was that if communicated in a PIN / as a PIN, the preliminary market consultation would never be combined with a call for competition (because the consultation would always come earlier). Would you agree? 

(Reading the definition of a PIN from a directive, 2014/24/EU Art. 48, I'd say that "Contracting authorities may make known their intentions of planned procurements through the publication of a prior information notice." would cover also a special PIN for a preliminary market consultation. However, I don't have a particularly strong opinion.)   

@bouwesamplonius, concerning selection criteria, if the PIN is a "PIN only", this should not be a problem, because selection criteria are not required in this case. More generally, any fields that do cause these types of problems could be taken care of through business rules along the lines of "if the PIN is used as a PMC, then BT-XYZ is not mandatory". 

@MiraRatajczak, good point, this could be solved by adding another deadline BT or expanding one of the current ones, e.g. BT-131 (Deadline Receipt Tenders) - which already covers the submission of more than just tenders.

**Comment by MiraRatajczak** [comment]
A view on combining CFC after a PMC: 
In case you launch a Prelimary Market Consultation (PMC) and you demand a return of information in it. 
1.	Then the CFC is not interesting to use because you know who is interested in your procurement. 
2.	The PIN time limit is not interesting because you already have launched a PIN PMC so you can use the time limit reduction. For so far the publication happens between 35 days and a year.

We agree that we don’t create an extra field but reuse the BT-131. But can we accommodate the description of this field.

**Comment by bouwesamplonius** [comment]
Hi @JachymHercher,

The opinion of TenderNed and the other dutch tendering systems is that a new form only for PMC is the way to go. With a seperate form, it becomes clear for the interested party that is is not really a tender according to the regulations. What the interested party will be think when it is a PIN. After the PMC the decision will be made if a tender is going to start. 

With another sub-type of the PIN we only create more exceptions and business rules. We like to create it more simple. The PMC is becoming more and more popular.

Do you need a subset of fields what we like to be part of a seperate PMC form? I can discuss this with the other dutch tendering systems.

**Comment by rantati7** [comment]
@bouwesamplonius Finland supports your view!

**Comment by rantati7** [comment]
@bouwesamplonius Filnad supports your view. When revewing our eNotification system this is what we intend to do anyway: e selarate form for PMC (one for under thresholds and another for over thresholds that would be also published in TED (in PIN (with no intention to shorten the deadlines or in new PCM)

**Comment by JachymHercher** [comment]
Ok, we note that you prefer a separate notice, even though even more opinions would be welcomed. We will change the spreadsheet accordingly. 

@bouwesamplonius and others: yes, we would be grateful for a list of BTs (reusing those from eForms as far as possible) that you would like to have in a PMC form.

**Comment by bouwesamplonius** [comment]
I make a list @JachymHercher and review it the following week with the stakeholders. Maybe it is wise te remove the 'closed' label to get more attention?

**Comment by bouwesamplonius** [comment]
Hi @JachymHercher,

Here you have our selection of BTs for the PMC. A few considerations:

- In short: PMC is not a type given in the procurement directives. We don't want to give interested party's the idea that PMC is a official tender procedure or a guaranteed start of a procedure.

- Our opinion is that contracting authorities are creating a separate document in which they describe what they need for information from the market in the PMC. The most contracting authorities in The Netherlands are working like this. Because of that it is not necessary to put everything in the publication.

- A short description about the PMC in the publication. BT-24 can be used or the new BT-117. My advice is to not limit this field at a specific amount of characters. 

- Everything about awarding, exclusion grounds is not needed. 

- BT-04 Procedure Identifier is necessary i think to differentiate the PMC of the other forms? But PMC is not a official procedure according to the regulations. The same for BT-105. PMC is not part of the codelist but needs to be added.

- BT-131 can be used to communicate when the PMC is ended, but maybe it is more convenient to create a separate BT? 
[PMC.xlsx](https://github.com/eForms/eForms/files/2388313/PMC.xlsx)

**Comment by JachymHercher** [comment]
Thank you for the list, we will use it as a basis for the PMC notice.

Please note that since the PMC is not in the directive, we will implement this notice as an extension. As such, the detailed discussions are likely to be had only once the act is adopted. (Assuming that the legal act keeps this possibility.)

For the next version of the spreadsheet, which will be sent into the official adoption procedure, the only change will be the removal of BT-617 and BT-117, because they are no longer necessary - the PMC notice would be used instead.

**Comment by manevmi** [comment]
Also in favour of a separate PMC notice rather than using the PIN. 
In principle the PMC comes before the contracting authority decides to launch a procedure. After all, the result of the PMC can be that the market cannot satisfy the buyer's need in which case there is no sense to launch a procedure at all (or to announce the intention of launching one through a PIN).

**Comment by JachymHercher** [comment]
Since we are currently waiting for the final opinions from Member States, we have come back to the topic of notices that are not explicitly covered by the Directives. Such notices can be published in TED in line with Art. 51 of Directive 2014/24/EU. 

For the prior market consultation form, based on the discussions above and looking at other sources (e.g. the slightly unhelpful p.24 of the [public procurement guidance](https://ec.europa.eu/regional_policy/sources/docgener/guides/public_procurement/2018/guidance_public_procurement_2018_en.pdf)), we would suggest it contains the terms listed below. All are re-used from the eForms proposal. 

There are quite a few, but, similarly to eForms, those marked as optional do not need to be used at all, so in the end the PMC can be quite short. We would be grateful for any comments.

ID | Name | Data type | Repeatable | Description|PMC
-- | -- | -- | -- | -- | --
BG-1 | Notice | - | No | Basic information about the   notice. | M
BT-701 | Notice Identifier | Identifier | No | The European   Public Procurement Notice Identifier of this notice. Including this   identifier in all published version of this notice (e.g. TED, national   publication portals, regional publication portals) allows unique   identification of procurement notices around the EU. | M
BT-757 | Notice Version | Identifier | No | The version of   the notice. This helps, for example, to avoid errors caused by multiple   change notices sent at a similar time. | M
BT-01 | Procedure Legal Basis | Code | No | The legal basis   (e.g. a European Directive or Regulation, national law) under which the   procurement procedure takes place. | M
BT-03 | Form Type | Code | No | The type of form   according to procurement legislation. | M
BT-02 | Notice Type | Code | No | The type of   notice according to procurement legislation. | M
BT-05 | Notice Dispatch Date | Date | No | The date and   time when the notice has been sent to TED. | M
BT-738 | Notice Publication Date Preferred | Date | No | The preferred   date of publication of the notice on TED (e.g. to avoid publication during a   national holiday). | O
BT-702 | Notice Official Language | Code | Yes | The language(s)   in which this notice is officially available. These linguistic versions are   equally legally valid. | M
BG-125 | Previous   Planning | - | Yes | Information about   a prior information notice, a periodic indicative notice or another similar   notice related to this notice. This information may differ per lot. In case   of a prior information notice or a periodic indicative notice used only for   information, this information may differ per part of the notice that may   later become a lot or a self-standing procedure. | O
BT-1371 | Previous Planning Lot Identifier | Identifier | Yes | An identifier of   one or more lots. In case of PINs used only for information, the identifier   of a part of the notice that may later become a lot or a self-standing   procedure. The information in the previous planning section refers to this   lot (or part) or these lots (or parts). | CM
BT-125 | Previous Planning Identifier | Identifier | No | The identifier   of a prior information notice, a periodic indicative notice or another   similar notice related to this notice. | O
BT-1251 | Previous Planning Part Identifier | Identifier | Yes | The identifier   of a part of a prior information notice, a periodic indicative notice or   another similar notice related to this notice. | O
BG-703 | Organisation | - | Yes | Information about   the organisation. This information may differ per lot, per contract, per   winning tender, etc. In case of a prior information notice or a periodic   indicative notice used only for information, this information may differ per   part of the notice that may later become a lot or a self-standing procedure. | M
BT-13720 | Organisation Notice Section Identifier | Identifier | Yes | An identifier of   one or more sections within this notice. The information in the organisation   section refers to this section or these sections. | CM
BT-500 | Organisation Name | Text | No | The official   name of the organisation. | CM
BT-501 | Organisation Identifier | Identifier | Yes | An identifier of   the organisation. All of the organisation's identifiers shall be given. | O
BT-16 | Organisation Part Name | Text | No | The name of a   part of an organisation (e.g. the relevant department of a large buyer). | O
BT-510 | Organisation Street | Text | No | The name of the   street, road, avenue, etc., of the organisation's physical address and   further identification (e.g. building number). | O
BT-513 | Organisation City | Text | No | The name of the   locality (city, town or village) of the organisation's physical address. | O
BT-512 | Organisation Post Code | Text | No | The post code of   the organisation's physical address. | O
BT-507 | Organisation Country Subdivision | Code | No | The location   according to the common classification of territorial units for statistics   (NUTS) of the organisation's physical address. The NUTS3 classification code   must be used. | O
BT-514 | Organisation Country Code | Code | No | The country of   the organisation's physical address. | CM
BT-502 | Organisation Contact Point | Text | No | The name of the   department or other contact point for communicating with the organisation. To   avoid unnecessary processing of personal data, the contact point may allow   the identification of a physical person only when necessary (in the sense of   Regulation (EU) 2016/679 and Regulation (EU) 2018/1725). | O
BT-506 | Organisation Contact Email Address | Text | No | The email for   contacting the organisation. To avoid unnecessary processing of personal   data, the email may allow the identification of a physical person only when   necessary (in the sense of the General Data Protection Regulation). | O
BT-503 | Organisation Contact Telephone Number | Text | No | The telephone   for contacting the organisation. To avoid unnecessary processing of personal   data, the telephone may allow the identification of a physical person only   when necessary (in the sense of the General Data Protection Regulation). | O
BT-739 | Organisation Contact Fax | Text | No | The fax for   contacting the organisation. To avoid unnecessary processing of personal   data, the fax may allow the identification of a physical person only when   necessary (in the sense of the General Data Protection Regulation). | O
BT-505 | Organisation Internet Address | URL | No | The website of   the organisation. | O
BT-509 | Organisation eDelivery Gateway | URL | No | The   organisation's uniform resource locator for exchange of data and documents. | O
BT-08 | Organisation Role | Code | No | The role of the   organisation in the procurement procedure (e.g. buyer, winner). A notice must   contain all organisations in the procedure that have the role of a buyer; a   winner; a review body; a buyer acquiring supplies and/or services intended   for other buyers; a buyer awarding public contracts or concluding framework   agreements for works, supplies or services intended for other buyers. | M
BT-770 | Organisation Subrole | Code | Yes | The sub-role of   the organisation in the procurement procedure (e.g. group leader,   organisation providing additional information about the procurement   procedure). | O
BG-3 | Buyer | - | No | Additional   information about the buyer. | O
BT-508 | Buyer Profile URL | URL | No | The website   where the buyer publishes information regarding procurement procedures (e.g.   notices, procurement documents). | O
BT-11 | Buyer Legal Type | Code | No | The type of   buyer according to procurement legislation (e.g. central government   authority, body governed by public law, public undertaking). | O
BT-740 | Buyer Contracting Entity | Indicator | No | The buyer is a   contracting entity. | O
BT-10 | Activity Authority | Code | No | The main   activity of the contracting authority. | O
BT-610 | Activity Entity | Code | No | The main   activity of the contracting entity. | O
BG-2 | Purpose | - | Yes | Information about the purpose.   This information must be given for the whole procurement procedure and, if   they exist, also per lot. In case of a prior information notice or a periodic   indicative notice used only for information, this information may differ per   part of the notice that may later become a lot or a self-standing procedure. | M
BT-137 | Purpose Lot Identifier | Identifier | No | The identifier   of a lot. In case of PINs used only for information, the identifier of a part   of the notice that may later become a lot or a self-standing procedure. The   information in the purpose section refers to this lot or part. | CM
BT-22 | Internal Identifier | Text | No | The internal   identifier used for files regarding the procurement procedure or lot before a   procedure identifier is given (e.g. coming from the buyer's document   management system or procurement planning system). | O
BT-23 | Main Nature | Code | No | The main nature   (e.g. works) of what is being bought. In case of mixed procurement (e.g. a   procedure for both works and services), the main nature may be, for example,   the one with the highest estimated value. This information must be given for   the whole procedure. | CM
BT-531 | Additional Nature | Code | Yes | The nature (e.g.   services) of what is being bought, additional to Main Nature. | EM
BT-21 | Title | Text | No | The name of the   procurement procedure or lot. | M
BT-24 | Description | Text | No | The description   of the nature and quantity of what is being bought or of the needs and   requirements that should be met in this procedure or lot. In case of a   modification notice, the description of the procurement before and after the   modification. | M
BT-27 | Estimated Value | Value | No | The estimated   maximum value of the procurement procedure or lot. Estimated refers to   estimation at the time of launching the call for competition. Maximum value   means a value covering all contracts to be awarded within a framework   agreement or a dynamic purchasing system, over their whole duration,   including options and renewals. | O
BG-557 | Group Framework Estimated Maximum Value | - | Yes | Information about   the estimated maximum value which can be spent in a framework agreement   within a group of lots. This information can be provided when the estimated   maximum value of a group of lots is lower than the sum of estimated values of   individual lots in this group (e.g. when the same budget is shared for   several lots). | O
BT-557 | Group Framework Estimated Maximum   Value Lot Identifier | Identifier | Yes | Identifiers of   lots. These lots form a group whose estimated maximum value is lower than the   sum of individual estimated maximum values of all the lots together (e.g.   when the same budget is shared for several lots). | O
BT-157 | Group Framework Estimated Maximum   Value | Value | No | The estimated   maximum value which can be spent in a framework agreement within a group of   lots. This information can be provided when the estimated maximum value of a   group of lots is lower than the sum of estimated values of individual lots in   this group (e.g. when the same budget is shared for several lots). Estimated   refers to estimation at the time of launching the call for competition.   Maximum value means a value covering all contracts to be awarded within a   framework agreement or a dynamic purchasing system, over their whole   duration, including options and renewals. | O
BG-261 | Classification | - | Yes | Information about   the classification(s) describing the purchase. The common procurement   vocabulary (CPV) classification type must be used. Additionally, other   classifications (e.g. the World Health Organisation's anatomical therapeutic   chemical (ATC) classification for pharmaceuticals; the medical devices   nomenclature under the Regulation (EU) 2017/745) may, if made available by   the Publications Office of the EU, be used.) | M
BT-26 | Classification Type | Code | No | The type of   classification describing the purchase (e.g. the common procurement   vocabulary). | M
BT-262 | Main Classification Code | Code | No | The code from   the classification that best characterizes the purchase. | CM
BT-263 | Additional Classification Codes | Code | Yes | An additional   code from the classification that also characterize the purchase. | O
BG-708 | Place   of Performance | - | Yes | Information about the main   location of works in case of works; main place of delivery or performance in   case of supplies and services. If the place of performance covers several   NUTS 3 areas (e.g. a highway, a national network of job centres), then all relevant   codes must be given. This information may differ per lot. In case of a prior   information notice or a periodic indicative notice used only for information,   this information may differ per part of the notice that may later become a   lot or a self-standing procedure. | O
BT-1372 | Place Lot Identifier | Identifier | Yes | An identifier of   one or more lots. In case of PINs used only for information, the identifier   of one or more parts of the notice that may later become lots or a   self-standing procedures. The information in the place of performance section   refers to this lot (or part) or these lots (or parts). | CM
BT-5101 | Place Street | Text | No | The name of the   street, road, avenue, etc., of the place of performance and further   identification (e.g. building number). | O
BT-5131 | Place City | Text | No | The name of the   locality (city, town or village) of the place of performance. | O
BT-5121 | Place Post Code | Text | No | The postcode of   the place of performance. | O
BT-5071 | Place Country Subdivision | Code | No | The location   according to the common classification of territorial units for statistics   (NUTS). The NUTS3 classification code must be used. | O
BT-5141 | Place Country Code | Code | No | The country of   the place of performance. | O
BT-727 | Place Services Other | Code | No | There are other   restrictions on the place of performance (e.g. "anywhere in the European   Economic Area", "anywhere in the given country"). | O
BT-728 | Place Additional Information | Text | No | Additional   information about the place of performance. | O
BG-36 | Duration | - | Yes | Information about   the duration of the contract, framework agreement, dynamic purchasing system   or qualification system. This must include any options and renewals. This   information may differ per lot. In case of a prior information notice or a   periodic indicative notice used only for information, this information may   differ per part of the notice that may later become a lot or a self-standing   procedure. | O
BT-1373 | Duration Lot Identifier | Identifier | Yes | An identifier of   one or more lots. In case of PINs used only for information, the identifier   of one or more parts of the notice that may later become lots or a   self-standing procedures.  The   information in the duration section refers to this lot (or part) or these   lots (or parts) | CM
BT-536 | Duration Start Date | Date | No | The (estimated)   date when the contract, framework agreement, dynamic purchasing system or   qualification system will start. | O
BT-36 | Duration Period | Duration | No | The (estimated)   period from the start to the end of the contract, framework agreement,   dynamic purchasing system or qualification system. This must include any   options and renewals. | O
BT-537 | Duration End Date | Date | No | The (estimated)   date when the contract, framework agreement, dynamic purchasing system or   qualification system will finish. | O
BT-538 | Duration Other | Code | No | The duration is   unknown, unlimited, etc. | O
BG-100 | Communication | - | Yes | General information about   communication with the buyer(s). This information may differ per lot. In case   of a prior information notice or a periodic indicative notice used only for   information, this information may differ per part of the notice that may later   become a lot or a self-standing procedure. | O
BT-724 | Atypical Tool | Indicator | No | Electronic communication   requires the use of tools and devices that are not generally available. | O
BT-124 | Atypical Tool URL | URL | Yes | Electronic   communication requires the use of tools and devices that are not generally   available. The uniform resource locator (e.g. the web address) which gives   unrestricted and full direct access to these tools and devices. | O
BT-127 | Future Notice | Date | No | The estimated   date of publication of a contract notice within this procedure. | O
BG-101 | Procurement   Documents | - | Yes | Information about the procurement   documents. This information may differ per lot. In case of a prior   information notice or a periodic indicative notice used only for information,   this information may differ per part of the notice that may later become a lot   or a self-standing procedure. | O
BT-15 | Documents URL | URL | No | The internet address for   accessing (the non-restricted part of) the procurement documents. For all   notices except prior information notices used only for information and   periodic indicative notices used only for information, the address must give   access that is direct (i.e. the exact webpage with the documents, not a   general website), unrestricted (e.g. no payment, no registration) and full   (i.e. the procurement documents must be complete) and the documents must be   available already at the time of the publication of the notice. | O
BT-708 | Documents Official Language | Code | Yes | The language(s)   in which the procurement documents are officially available. These linguistic   versions are equally legally valid. | O
BT-737 | Documents Unofficial Language | Code | Yes | The language(s)   in which the procurement documents (or their parts) are unofficially   available. These linguistic versions are not an official translation, they   are provided only for information. | O
BT-14 | Documents Restricted | Indicator | No | The access to   certain procurement documents is restricted. | O
BT-707 | Documents Restricted Justification | code | No | The   justification for restricting access to certain procurement documents. | O
BT-615 | Documents Restricted URL | URL | No | The internet   address with information on accessing the restricted (part of the)   procurement documents. | O
BT-13 | Additional Information Deadline | Date | No | The time limit   for requesting additional information about the procurement procedure. | O
BG-710 | Additional   Information | - | Yes | Any information not mentioned   elsewhere. This information may differ per lot. In case of a prior   information notice or a periodic indicative notice used only for information,   this information may differ per part of the notice that may later become a   lot or a self-standing procedure. | O
BT-300 | Additional Information | Text | No | Any information not mentioned   elsewhere. | O

**Comment by mdewinne** [comment]
Hi Jachym, thank you for this interesting proposal, which certainly caters for a lot of our wishes.

A few remarks / questions from our side:  

- What is the purpose of having BG-125, allowing to refer to a PIN (or other) notice? From our point view, it seems that the PMC would practically always be the first notice in a series of notices (market prospection, then PIN, followed by a CN and finally a CAN).

- In relation to the previous remark, shouldn’t BG-127 “future notice” also allow to refer to a PIN (and not only to a CN)?

- Why is BG-708 “place of performance” optional? For a CA it should not be too difficult to pinpoint a location where the goods/services/works will be delivered/performed/executed. On the other hand, this information can be quite important for an EO participating in the market survey.

- We feel that the CA should be able to specify when and where the answers to the market survey should be submitted. We would suggest adding a field similar to a tender submission deadline and a URL (electronic platform).

**Comment by JachymHercher** [comment]
@mdewinne, thank you very much for the feedback. 

1. At some point, we received feedback (from the Netherlands, if I remember correctly, FYI @bouwesamplonius), that there could be multiple rounds of PMCs. BG-125 could be used to link one PMC to another.  
2. We saw the purpose of BT-127 as informing about the date when the procedure will be launched. This could be relevant for businesses, because they may be able to offer different things in different moments. (E.g. if it's clear that the procedure will be launched in a year, then some investment into innovative technologies may already be in place). In principle, we're not against referring also to PINs, but we're not sure how important this would be for companies?
3. Since this is a voluntary notice, we are really trying to keep it as light as possible. We agree that typically this would be useful information, but we would rather leave this decision to national systems. 
4. Good point. For the URL, we would suggest reusing BT-18 (Submission URL) and BT-97 (Submission Language) - perhaps with a slightly updated description. For the deadline, we would suggest adding a new field: 

BT-800 | Deadline Receipt Answers | Date | No | The time limit for receipt of   answers to the preliminary market consultation.
-- | -- | -- | -- | --

**Comment by mdewinne** [comment]
@JachymHercher 

Thank you for you comments.

1. We agree that it might be useful to be able to link different PMC notices.
2. You are right and we agree :)
3. Ok, this is consistent with the policy for many other fields
4. The proposed solutions suits us perfectly!

**Comment by bouwesamplonius** [comment]
@JachymHercher We still have your proposal in review but at first glance it is matching the terms and fields that we suggested. So I don't expect any negative comments.

@mdewinne 

1. Multiple rounds of PMCs are very common in The Netherlands, that is why we suggested to add BG-125.

2. I don't see very often a PMC followed-up by a PIN, but it is possible in for example TenderNed.

3. Agree with BT-800 as long it is a optional field. It is for a CA not always possible to define a time limit.

**Comment by JachymHercher** [comment]
@bouwesamplonius could you please elaborate a bit on cases when it wouldn't be possible (or practical) for a CA to define a time limit? In particular, if a time limit isn't specified, isn't there a danger of companies wasting resources on preparing their answers and sending them when the CA has already made a decision (and so it won't use the information anymore)? 

Related issue: for each notice, we need to define for how long it should be available on TED in the "current notices" tab. Calls for competition, for example, are included until the deadline for submitting tenders has passed. For PMC, a natural approach would be to leave them published until the deadline for receiving answers has passed. However, if there is no deadline for asking questions, this wouldn't be possible. 

Also, please keep in mind that if the CA gives a deadline, it can still change it later on with a change notice.

**Comment by ColinMaudry** [comment]
Hi everyone,

Could someone please spell out the following acronyms?

- PMC
- CA

Thanks!

**Comment by guascce** [comment]
CA = Contracting Authority
PMC ??

**Comment by PaulBoroday** [comment]
Preliminary Market Consultations

**Comment by bouwesamplonius** [comment]
@JachymHercher Just asked some CA's why they do not include a closing date for PMC. Usually this has to do with that the PMC is an invitation for a informal meeting about the tender. The CA just don't know if and when the PMC will continue after this meeting. We prefer for making BT-800 optional but are not against making it mandatory. What is common in Belgium @mdewinne?
